### ISSUE TITLE
.NET Core 3.0 and plugin scenario - Assembly.LoadFrom fails when an older version of dependency assembly has already been loaded

### ISSUE THREAD

**Issue by sdg002** [dotnet/corefx#41276]
Hi All,
I had initially reported this issue under DOTNET/CORE repository. Since, I did not receive any response, I though I should verify if this issue belongs to DOTNET/COREFX.

**Original issue**
https://github.com/dotnet/core/issues/3426

I am happy to re-submit this issue here.

Thanks,
Sau

**Comment by jkotas** [comment]
> Assembly.LoadFrom fails when an older version of dependency assembly has already been loaded

Yes, this behavior is by design in .NET Core. Loading multiple versions of the same assembly side-by-side is supported by `Assembly.LoadFile` or `AssemblyLoadContext` APIs only in .NET Core.
